### DISCUSSION TITLE
use /tools/convert_dlib_nets_to_caffe convert 'shape_predictor_68_face_landmarks.dat' to caffe.

### DISCUSSION THREAD

**Discussion by Toasting-bit** [davisking/dlib#2901] [Q&A]
"Using '/tools/convert_dlib_nets_to_caffe' to convert 'shape_predictor_68_face_landmarks.dat' to caffe cannot be successful. Has anyone succeeded, or is it not possible to convert 'shape_predictor_68_face_landmarks.dat' using the '/tools/convert_dlib_nets_to_caffe' command?"

**Comment by arrufat** [comment]
It's not possible, that tool can only convert a subset of dlib's neural networks.
The Shape Predictor, however, is not a neural network.
